scripts/sampler/evaluate.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+from openai.types.chat import (
+    ChatCompletionAssistantMessageParam,
+    ChatCompletionUserMessageParam,
+)
+
+from mentat.errors import SampleError
+from mentat.git_handler import get_git_diff
+from mentat.python_client.client import PythonClient
+from mentat.sampler.utils import get_active_snapshot_commit, setup_repo
+from mentat.session_context import SESSION_CONTEXT
+
+
+async def evaluate_sample(sample, cwd: Path | str | None = None):
```

Comment:
What do you think about making this one an instance method of samples? I'd like to use it in the benchmark runner itself and it'd be easy to get there.